File: php/src/SudokuValidator.php
<?php

namespace TwentyFiveFriday;

class SudokuValidator
{
    /**
     * @param string[][] $sudoku
     */
    public function __construct(private array $sudoku) {}
    
    public function validate(): string {
        return 'you should return "Valid" or "Not Valid"';
    }
}

File: Dotnet/Friday.Sudoku/src/Friday.Sudoku/SudokuValidator.cs
namespace Friday.Sudoku;

public class SudokuValidator
{
    private string[][] _sudokuMatrix;
    
    public SudokuValidator(string[][] sudokuMatrix)
    {
        _sudokuMatrix = sudokuMatrix;
    }
    
    public string Evaluate()
    {
        return "you should return Valid or Not Valid";
    }
}